contents/handbook/growth/sales/crm.md

```diff
@@ -81,28 +81,52 @@ You can also just manually add a user to HubSpot under 'Contacts'. When creating
-We use [Clearbit](https://clearbit.com/) to enhance our contact information as they are created and then compute a score out of 5 based on the following parameters:
+We use [Clearbit](https://clearbit.com/) to enhance our contact information as they are created and then compute a score out of 25 in HubSpot based on the following parameters:
 
-- *Industry* - customers in certain industries have stricter privacy requirements which mean a privacy-focused solution like PostHog is ideal for them.  We give the highest score to the following industries:
+- *Industry* - customers in certain industries have stricter privacy requirements which mean a privacy-focused solution like PostHog is ideal for them.  We give the highest score (5 points) to the following industries:
   - Financial Services
   - Health Care
   - Biotechnology
   - Health Care Services
   - Legal Services
   - Health & Wellness
   
-  Whilst we score them slightly lower, we also focus on customers from:
+  Whilst we score them slightly lower (4 points), we also focus on customers from:
   - Internet Software & Services
   - Consulting
   - Computer Networking
 - *Employee count* - we use this as a strong indicator for product market fit.  Smaller companies are less likely to have achieved this so our highest score here goes to companies in the 100-1000 employee range.  We score companies over 1000 employees slightly lower as they will generally be slower to deal with.
-- *Ability to pay* - indicates whether a company is likely to pay for a product like PostHog to solve their problems.  This is computed from a combination of company revenue and/or the amount of money they have raised.  If their revenue is over $10m, or they have raised over $20m they get the highest score.
-- *Role* - from experience we sell best to people in an engineering role and score those the highest.  We also do well with leadership and product folks, so they have a favourable score.
+- *Ability to pay* - indicates whether a company is likely to pay for a product like PostHog to solve their problems.  This is computed from a combination of company revenue and/or the amount of money they have raised.  If their revenue is over $10m, or they have raised over $20m they get the highest score (5 points). If revenue is under $1-10m or they've raised over $10m they'll get 4 points.
+- *Role* - from experience we sell best to people in an engineering role (5 points) and score those the highest.  We also do well with leadership (4 points) and product (3 points) folks, so they have a favourable score.  If we know the role but it falls outside of those categories then it's 1 point.
 - *B2B/B2C* - our event-based pricing works best for business-to-business companies where the value per event is higher.
+- *Country* - from experience we know that certain countries have a lower inclination to pay for software so we downweight those.
```

Comment:
I think it's better to be transparent here and state how we score countries? (Knowing this will help us target our marketing efforts for example, and I don't think we'll be 'insulting' other countries provided this is framed sensitively!)

Reply:
Especially given we apply a -7 to 'not in target country list', which is pretty chunky

Reply:
Added those now